composer.json

```diff
@@ -26,22 +26,24 @@
         "php": ">=5.3.6",
         "ext-zip": "*",
         "ext-fileinfo": "*",
-        "symfony/console": "~2.3",
-        "symfony/finder": "~2.2",
-        "symfony/filesystem": "~2.1",
-        "symfony/process": "~2.1",
+        "symfony/console": ">=2.5||~3.0",
```

Comment:
The `>=2.5` part would mean it's also compatible with 3.0 and a future 4.0.

It we keep compatibility with previous releases, this should probably be something like `~2.3 || ~3.0` or `^2.3 || ^3.0`, see e.g. composer itself: https://github.com/composer/composer/blob/876f14341814d427e87dd358ed91ccc855678a5c/composer.json#L32